cli/requirements.txt

```diff
@@ -1,5 +1,5 @@
 boto3>=1.16.14
-tabulate>=0.8.2,<0.8.10
+tabulate>=0.8.8,<=0.8.10
```

Comment:
Is there any specific reason to set the 0.8.10 as the last, what about `tabulate>=0.8.8,<0.9`

Reply:
In our code, we have the third party licenses for version `0.8.10`
```
https://github.com/aws/aws-parallelcluster/blob/develop/THIRD-PARTY-LICENSES.txt
```
Not sure if the licensing will change after 0.8.10. And we don't have version between 0.8.10 and 0.9 now